### PR TITLE
[feat]: contributing guideline 

### PR DESCRIPTION
**Pull request by poriacodes**
Fixes issue #1 

This contributing guideline was created to help project contributors learn the right process for contributing to this repository.

Kindly review. 

Thank you.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Contributing to The Sailscasts Docs
+
+We appreciate your interest in contributing to this docs! Your contributions are very valuable to us, and we want to ensure that the process is easy and productive for everyone involved. 
+
+Please take a moment to read the guidelines below before contributing to the project.
+
+## Project Description
+
+This project contains the official documentation of all the products built for The Sailscasts Community.
```
Comment (DominusKelvin) [inline CONTRIBUTING.md:9]: Can we make this "This project contains the official documentations for all projects both open source and commercial by The Sailscasts Company"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+#### These products includes:
```
Comment (DominusKelvin) [inline CONTRIBUTING.md:11]: I don't think we should list the projects as the list will get stale pretty quickly and needs update.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+In this workflow, we use the develop branch as the central branch for active development. Therefore, we request that all Pull Requests be targeted towards the develop branch instead of the main branch. This approach ensures the stability of our main branch while allowing continuous development on the develop branch.
+
+## Before Contributing
+
+Before diving into the contribution process and setting up your environment, it is essential to select an issue you would like to work on.
+
+#### Here's a simple guide to follow:
+
+1. Search through the list of [open issues](https://github.com/sailscastshq/docs.sailscasts.com/issues).
```
Comment (DominusKelvin) [inline CONTRIBUTING.md:32]: Nice, do you think we should mention the possibility of creating an issue if one doesn't exist for the PR you want to submit?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    This helps us track and automatically close the relevant issue when your Pull Request is merged.
+
+## Contribution Etiquettes
+
+- When you pick an issue to fix, we recommended that you review the comment thread first to check if someone is already working on it. In case no one has claimed it yet, kindly leave a comment stating your intention to work on it. This way, others will be aware, and we can avoid accidental duplication of effort.
+
+- If someone has shown interest in fixing an issue but doesn't follow up for a specific period, let's say 2-3 weeks, it's permissible to pick up the same issue later. However, please ensure that you leave a comment to inform others of your intention before proceeding.
+
+
+   #### Thank you for reading the guide. We look forward to merging your contributions.
```
Comment (DominusKelvin) [inline CONTRIBUTING.md:107]: We may need to check the indentation here.

**Review by DominusKelvin — CHANGES_REQUESTED** [review]
Great job @poriacodes this is looking good.

I left a couple of notes for you for some changes

File: CONTRIBUTING.md (hunk 1 above)
Comment (poriacodes) [reply CONTRIBUTING.md:9]: Done

File: CONTRIBUTING.md (hunk 2 above)
Comment (poriacodes) [reply CONTRIBUTING.md:11]: Alright

File: CONTRIBUTING.md (hunk 4 above)
Comment (poriacodes) [reply CONTRIBUTING.md:107]: Done

File: CONTRIBUTING.md (hunk 3 above)
Comment (poriacodes) [reply CONTRIBUTING.md:32]: I mentioned that in number 2 of "Before Contributing"

**Review by DominusKelvin — APPROVED** [review]

**Comment by DominusKelvin** [comment]
Great job on this one @poriacodes

**Pull request merged**